scraper_atlanta.py

```diff
@@ -0,0 +1,180 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from datetime import datetime, timedelta
+from enum import Enum
+# import time
+
+
+class TableHTML(Enum):
+    """
+    The class contains variables that describe the structure of the table to be scraped
+    """
+    HEADER_AND_ABOVE = 3  # table header and content above has 3 `tr` tags
+    PAGE_NAV = 2  # page navigation bar has 2 `tr` tags
+    CONTENT_AND_PAGE_NAV = 12  # maximum table content plus page navigation bar has 12 `tr` tags
+    HEADER_ROW_POSITION = 2  # the 3rd 'tr' tag in table tag
```

Comment:
I see header row is always the second `tr` element in `tbody` tag. `ACA_TabRow_Header ACA_TabRow_Header_FontSize` is the class name.

Reply:
There's another `tr` nested in the first `tr` actually.
I tried to use the class name for the header at the beginning, it says selenium doesn't support some thing called `compound class name` or something like that. I guess there's an space in the class name. And oftenly, i cannot locate the element by `xpath`  in selenium. That's why I used the absolute position to locate the element.